apps/metrics/__init__.py

```diff
@@ -0,0 +1,115 @@
+from common.utils import camel2snake
+
+
+class MetricType:
+    UNKNOWN = 'unknown'
+    TIME = 'time'
+    SIZE = 'size'
```

Comment:
Size is a bit broad I think. Bandwidth? Content-size? Something that will indicate a proper unit (kB, Bytes, etc.)